From 1d28c1813b313943012f9619d5b3852e5d994a57 Mon Sep 17 00:00:00 2001
From: Steffen Sledz <sledz@dresearch-fe.de>
Date: Wed, 20 Jan 2016 15:23:13 +0100
Subject: [PATCH] usb: gadget: composite: os descriptor specific fix

Add case USB_EXT_PROP_UNICODE_MULTI to fill_ext_prop.

Signed-off-by: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
Signed-off-by: Steffen Sledz <sledz@dresearch-fe.de>
---
 drivers/usb/gadget/composite.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/gadget/composite.c b/drivers/usb/gadget/composite.c
index 58b4657..4220381 100644
--- a/drivers/usb/gadget/composite.c
+++ b/drivers/usb/gadget/composite.c
@@ -1418,6 +1418,7 @@ static int fill_ext_prop(struct usb_configuration *c, int interface, u8 *buf)
 				case USB_EXT_PROP_UNICODE:
 				case USB_EXT_PROP_UNICODE_ENV:
 				case USB_EXT_PROP_UNICODE_LINK:
+				case USB_EXT_PROP_UNICODE_MULTI:
 					usb_ext_prop_put_unicode(buf, ret,
 							 ext_prop->data,
 							 ext_prop->data_len);
-- 
2.7.0

